detd/manager.py

```diff
@@ -96,17 +96,15 @@ def add_talker(self, config):
 
 class InterfaceManager():
 
-    def __init__(self, interface, options):
+    def __init__(self, interface, hints):
 
         logger.info(f"Initializing {__class__.__name__}")
 
         self.interface = interface
-        self.options = options
+        self.hints = hints
```

Comment:
Here the InterfaceManager needs to check the content of the provided hints with the device and platform features.

Each device will have a default hint based on its capabilities. We can provide a generic method and then let each device override it. I am adding it to https://github.com/Avnu/detd/issues/21

Some checks to perform are detailed in task "provide a default parameter flow until InterfaceManager", at https://github.com/Avnu/detd/issues/21